packages/api-gateway-service/service.ts

```diff
@@ -71,6 +76,16 @@ stitchedSchemas()
           Authorization: `Bearer <ENTER_API_KEY_HERE>`,
```

Comment:
Can we use this ```Bearer <API KEY>``` as environment variable to avoid this error.

Reply:
Which error?

Reply:
Added a comment on it to suppress the warning as it is a false positive.